Source/Cogworks.Examine.Tweaks/Configurations/CustomIndexConfig.cs

```diff
@@ -21,7 +21,7 @@ IContentValueSetValidator IUmbracoIndexConfig.GetContentValueSetValidator()
             => !(!TweaksConfiguration.InternalIncludedItemTypes.HasAny()
                     && !TweaksConfiguration.InternalExcludedItemTypes.HasAny())
                 ? new ContentValueSetValidator(
-                    publishedValuesOnly: true,
```

Comment:
Also good spot 👍

Reply:
Maybe we can add it also as configuration ? 🤔

Reply:
@aochmann I guess PublishedContentSetValidator should index published content and ContentSetValidator should index all.

Reply:
Ok so we can merge it, good work and findings 💪